src/Domain/Creators/RegisterDerivedWithCharm.cpp

```diff
@@ -20,69 +20,68 @@
 namespace domain {
 namespace creators {
 namespace DomainCreators_detail {
-using Affine = CoordinateMaps::Affine;
-using Affine2D = CoordinateMaps::ProductOf2Maps<Affine, Affine>;
-using Affine3D = CoordinateMaps::ProductOf3Maps<Affine, Affine, Affine>;
-using Equiangular = CoordinateMaps::Equiangular;
-using Equiangular2D = CoordinateMaps::ProductOf2Maps<Equiangular, Equiangular>;
+// alias to CoordinateMaps namespace:
+namespace Maps = CoordinateMaps;
```

Comment:
This seems like a good use of `using namespace`.  (I'd be even happier if this could all go away through some metaprogramming as is being discussed elsewhere in this PR, but I don't have anything to add to that discussion right now.)